src/meta/service/src/network.rs

```diff
@@ -133,18 +87,42 @@ impl Network {
         }
     }
 
-    pub fn with_backoff_policy(mut self, policy: BackOffPolicy) -> Self {
+    /// Set exponential back off policy for meta service
+    ///
+    /// - `ratio`: delay increase ratio of meta
+    ///
+    ///   should be not smaller than 1.0
+    ///
+    /// - `min_delay`: minimum back off duration, where the backoff duration vary starts from
+    ///
+    ///   random jitter between [0, min_back) will be set in each backoff, to minimize conflicts.
+    ///
+    /// - `max_delay`: maximum back off duration, if the backoff duration is larger than this, no backoff will be raised
+    ///
+    /// - `chances`: maximum back off times, chances off backoff
+    pub fn with_back_off_policy(
```

Comment:
Why do we need to store `ExponentialBackoff` in the `Network` struct? It's not a reuseable struct.